.ahoy.yml

```diff
@@ -104,13 +99,29 @@ commands:
     cmd: |
       ahoy confirm "Running this command will replace your current database. Are you sure?" &&
       # Run this if confirm returns true
-      cat .env | grep MARIADB_DATA_IMAGE | cut -c20- | xargs -n1 docker pull && docker-compose rm -f -s -v mariadb && ahoy up ||
+      ( cat .env | grep MARIADB_DATA_IMAGE | cut -c20- | xargs -n1 docker pull; docker-compose rm -f -s -v mariadb && ahoy up ) ||
```

Comment:
Currently, when this command is called and we have the latest DB image locally, this command will not proceed with the refresh of the container. This fixes it.